Library/Homebrew/cask/audit.rb

```diff
@@ -133,6 +133,8 @@ def check_required_stanzas
     end
 
     def check_version_and_checksum
+      return if cask.sha256 == :no_check
```

Comment:
Do we really want to allow `:no_check` for any constellation? Or just for documented cases of known-volatile SHAs, such as https://github.com/Homebrew/homebrew-cask/pull/51795?

Reply:
I think we *already* allow it. This is just to skip the check when the checksum changes.

Reply:
> I think we already allow it.

Indeed. TIL!